bolt/lib/Passes/SplitFunctions.cpp

```diff
@@ -409,8 +487,10 @@ void SplitFunctions::splitFunction(BinaryFunction &BF, SplitStrategy &S) {
     LLVM_DEBUG(dbgs() << "Estimated size for function " << BF
                       << " post-split is <0x" << Twine::utohexstr(HotSize)
                       << ", 0x" << Twine::utohexstr(ColdSize) << ">\n");
-    if (alignTo(OriginalHotSize, opts::SplitAlignThreshold) <=
-        alignTo(HotSize, opts::SplitAlignThreshold) + opts::SplitThreshold) {
+    if (S.autoReversal() &&
```

Comment:
Do you mean to run `calculateEmittedSize()` above even when `autoReversal()` is off?

Reply:
Removed `autoReversal()` per our offline discussion.